apps/meteor/app/file-upload/server/lib/FileUpload.ts

```diff
@@ -308,9 +308,20 @@ export const FileUpload = {
 		const store = FileUpload.getStore('Uploads');
 		const image = await store._store.getReadStream(file._id, file);
 
-		const transformer = sharp().resize({ width, height, fit: 'inside' });
+		let transformer = sharp().resize({ width, height, fit: 'inside' });
 
-		const result = transformer.toBuffer({ resolveWithObject: true }).then(({ data, info: { width, height } }) => ({ data, width, height }));
+		if (file.type === 'image/svg+xml') {
+			transformer = transformer.png();
+			if (file.name) {
+				file.name = `${file.name}.png`;
+			}
```

Comment:
Is there any reason we need to change the file name? 🤔 This can lead to undesired behaviors like `filename.svg.png`
Would it be a problem to keep the filename as is?

If this is extremely necessary, let's return a new property then, like `thumbFilename` instead, it would be clearer.

Reply:
Hey @MarcosSpessatto  When I download the image on clicking "save image" in chrome, the thumb file is downloaded which will have `.svg` extension. The user computer will try to open this image as an svg file whereas it is a PNG file.
Therefore I attached a .png file.

Reply:
hey @MarcosSpessatto I found that it was not necessary to add extension here. So I have removed  code to change filename.